src/lib/api/index.ts

```
@@ -1,3 +1,7 @@
 // Add any exports from your API files here
 
 export { getExample } from "./getExample";
+export {
+  updateVolunteer,
+  validateVolunteerUpdateBody,
```

We only want to export `updateVolunteer` function. Can we validate the inputs/body inside the function before creating the Supabase client? We want to return an error if the input/body is wrong.